src/runtime/hack/fc_metrics/README.md

```diff
@@ -0,0 +1,9 @@
+## fc-metrics-generator
+
+`fc-metrics-generator` is used to convert from Firecracker's metrics.rs to golang version of metrics.
+
+```
+update-generated-fc-metrics https://raw.githubusercontent.com/firecracker-microvm/firecracker/master/src/logger/src/metrics.rs
+```
+
+The generated golang source codes will be saved to file `src/runtime/virtcontainers/fc_metrics.go`.
```

Comment:
How about making it `src/runtime/virtcontainers/generated/fc_metrics.go` to make it clearer and avoid us potentially having to hardcode `fc_metrics.go` in a blocklist to exclude it (since it is auto-generated and effectively "read-only").